index.yaml

```diff
@@ -54,6 +54,18 @@ distributions:
       ubv8: foxy/release-focal-arm64-build.yaml
     source_builds:
       default: foxy/source-build.yaml
+  galactic:
+    ci_builds: {}
+    doc_builds: {}
+    notification_emails:
+      - scott+build.ros2.org@openrobotics.org
+      - ros2-buildfarm-rolling@googlegroups.com
```

Comment:
Should we set up the new Google group for Galactic builds?

Reply:
Done: https://groups.google.com/g/ros2-buildfarm-galactic